src/api/spec/components/bs_request_activity_timeline_component_spec.rb

```diff
@@ -1,13 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe BsRequestActivityTimelineComponent, type: :component do
-  context 'when we provide a bs_request' do
-    it 'shows an element telling who created the request and when'
-    it 'shows the comments threads'
-    it 'shows the history elements'
+  context 'when we do not provide a bs_request' do
+    it do
+      expect { render_inline(described_class.new) }.to raise_error(ArgumentError, 'missing keyword: :bs_request')
```

Comment:
I don't really see the benefit in testing how keyword arguments work. This is a core Ruby feature. I suggest removing this test.

Reply:
You are right. Changes pushed. I'm going to delete it from the other specs I added too.